static/templates/confirm_dialog/confirm_reactivate_user.hbs

```diff
@@ -1,8 +1,6 @@
 <p>
     {{#tr}}
-    <z-user></z-user> will have the same role, stream subscriptions,
-    user group memberships, and other settings and permissions as they did
-    prior to deactivation.
+    <z-user></z-user> will have the same role, stream subscriptions, user group memberships, and other settings and permissions as they did prior to deactivation.
```

Comment:
Why make this change?

Reply:
Please revert the changes in this file; the original line wrapping was preferred, and the error was about trailing whitespace you'd added in a previous commit, not the original text.

(Trailing whitespace is extra spaces after the end of the visible text on a line).